client/my-sites/stats/all-time/index.jsx

```diff
@@ -94,7 +94,7 @@ class StatsAllTime extends Component {
 						<StatsTab
 							className="all-time__is-best"
 							gridicon="trophy"
-							label={ translate( 'Best Views Ever' ) }
+							label={ translate( 'Best views ever' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.